docs/features/decorators.rst

```diff
@@ -9,7 +9,7 @@ Modules
 -------
 
 Robottelo decorators are located under
-:doc:`decorator package </api/robottelo.decorators>`. Most of them are used to
+:doc:`decorator package </autoapi/robottelo/decorators>`. Most of them are used to
```

Comment:
Apparently, it must be like below, or build fails:
```suggestion
:doc:`decorator package </autoapi/robottelo/decorators/index>`. Most of them are used to
```

Reply:
Thank you, I was wondering why build has failed :smile: